src/tests/ftest/control/ssd_socket.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+from apricot import TestWithServers
+from general_utils import run_command, DaosTestError
+
+
+class SSDSocketTest(TestWithServers):
+    """Test Class Description: Verify NVMe NUMA socket values.
+
+    Call dmg storage scan --verbose to obtain NUMA socket value (Socket ID) of
+    each NVMe disk. Verify against the value in
+    /sys/bus/pci/devices/<PCI address>/numa_node
+
+    :avocado: recursive
+    """
+
+    def test_scan_ssd(self):
+        """
+        JIRA ID: DAOS-3584
+
+        Test Description: Verify NVMe NUMA socket values.
+
+        :avocado: tags=all,small,full_regression,hw,control,ssd_socket
+        """
+        # Call dmg storage scan --verbose and get the PCI addresses.
+        data = self.get_dmg_command().storage_scan(verbose=True)
+        pci_addrs = data[self.hostlist_servers[0]]["nvme"].keys()
+        self.log.info("Testing PCI addresses: {}".format(pci_addrs))
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments